apps/fz_http/lib/fz_http_web/router.ex

```diff
@@ -15,9 +15,6 @@ defmodule FzHttpWeb.Router do
     plug :put_root_layout, {FzHttpWeb.LayoutView, :root}
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-
-    # XXX: Make this configurable
-    plug Hammer.Plug, @root_rate_limit
```

Comment:
I think you need to also remove this attribute:

https://github.com/firezone/firezone/actions/runs/3767295825/jobs/6405527115#step:16:289